enhancements/network/nodeport-range-configuration.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Make Service NodePort Range Configurable
+authors:
+  - "@abhat"
+reviewers:
+  - "@dcbw"
+  - "@deads2k"
+  - "@smarterclayton"
+  - "@danwinship"
+  - "@knobunc"
+approvers:
+  - "@knobunc"
+creation-date: 2020-07-06
+last-updated: 2020-07-06
+status: implementable
+---
+# Configurability of Service NodePort Range
+This enhancement targets making the service node-port range of an Openshift cluster configurable.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Service node-port range is a [configurable parameter](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/v4.1.0/config/defaultconfig.yaml#L147)
+on the Kubernetes API server. As our customers scale the number of node-port services in their Openshift clusters,
+there is a desire to make this node port range configurable.
+
+## Motivation
+
+Service node-port range is a configurable parameter consumed by the Kubernetes API server as
+a slice of port ranges. It defaults currently to a single port range (30000-32767).
```

Comment:
Is realistic having a cluster with 2767 node port services?

Reply:
yes, we have some customers who are already running into these issues.

Reply:
wow, awesome :muscle: